index.html

```diff
@@ -117,8 +117,74 @@ <h4 class="section3-heading">Proffesional Art Printing Data</h4>
     <a href="#" class="aboutme-button">Get My Resume</a>
-    <img class="vector-line" src="images/Vector4.png" alt="images">
-  </section>
+    <div class="side-image"></div>
+    <hr class="vector-line">
+    <div  class="aboutme-Languages">
+      <img class="diagonal-img" src="images/Diagonal.png" alt="images">
+      <p class="languages-1">Languages</p>
+    </div>
+    <ul class="aboutme-items">
+      <li class="aboutme-item">Javascript</li>
+      <li class="aboutme-item">Ruby</li>
+      <li class="aboutme-item">HTML</li>
+      <li class="aboutme-item">CSS</li>
+    </ul>
+    <div  class="aboutme-frameworks">
+      <img class="rectangle-img" src="images/Rectangle.png" alt="images">
+      <p class="frameworks-1">Frameworks</p>
+    </div>
+    <ul class="aboutme-frameworks-items">
+      <li class="aboutme-frame-item">Bootstrap</li>
+      <li class="aboutme-frame-item">Ruby on Rails</li>
+      <li class="aboutme-frame-item">RSPec</li>
+      <li class="aboutme-frame-item">CapyBara</li>
+      <li class="aboutme-frame-item">Selenium</li>
+    </ul>
+    <div  class="aboutme-skills">
+      <img class="circle-img" src="images/Circle.png" alt="images">
+      <p class="skills-1">Skills</p>
+    </div>
+    <ul class="aboutme-skills-items">
+      <li class="aboutme-skills-item">Bootstrap</li>
+      <li class="aboutme-skills-item">Ruby on Rails</li>
+      <li class="aboutme-skills-item">RSPec</li>
+      <li class="aboutme-skills-item">CapyBara</li>
+      <li class="aboutme-skills-item">Selenium</li>
+    </ul>
+ </section>
+ <p
+    class="last-paragraph">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <div class="inputs">
+      <p class="inputs-name">Full name</p>
+      <hr class="inputs-name-line">
+    <p class="iputs-email">Email address</p>
+    <hr class="inputs-email-line">
+  </div>
+  <textarea class="textarea" cols="30" rows="4">Enter text here</textarea>
+  <a href="#" class="close-button">Get in touch</a>
+  <hr class="sliceit-up">
+  <div>
+    <ul class="media-items">
+      <li>
+        <a href="#"><img class="media-item1" src="images/Github.png" alt="github"></a>
+      </li>
+      <li>
+        <a href="#"><img class="media-item2" src="images/Inner.png" alt="images-iner"></a>
+      </li>
+      <li>
+        <a href="#"><img class="media-item3" src="images/Peace.png" alt="media"></a>
+      </li>
+      <li>
+        <a href="#"><img class="media-item4" src="images/Twitter.png" alt="twitter-icon"></a>
+      </li>
+      <li>
+        <a href="#"><img class="media-item5" src="images/Mail.png" alt="mail-icon"></a>
+      </li>
+    </ul>
+  </div>
+  <div>
+    <img class="shaping-img" src="images/Shape.png" alt="images">
+  </div>
```

Comment:
- [ ] This section here is not part of the requirements of this project since it would be added in the next portfolio project. Do well to remove that.